package/react-native-mmkv.podspec

```diff
@@ -19,20 +18,15 @@ Pod::Spec.new do |s|
 
   s.pod_target_xcconfig = {
     "CLANG_CXX_LANGUAGE_STANDARD" => "c++17",
-    "CLANG_CXX_LIBRARY" => "libc++",
-    "CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF" => "NO",
-    # FORCE_POSIX ensures we are using C++ types instead of Objective-C types for MMKV.
-    "GCC_PREPROCESSOR_DEFINITIONS" => "$(inherited) FORCE_POSIX",
   }
   s.compiler_flags = '-x objective-c++'
   s.libraries    = "z", "c++"
   s.source_files = [
     # react-native-mmkv
     "ios/**/*.{h,m,mm}",
-    "cpp/**/*.{hpp,cpp,c,h}",
-    # MMKV/Core
-    "MMKV/Core/**/*.{h,cpp,hpp,S}",
+    "cpp/**/*.{hpp,cpp,c,h}"
   ]
 
+  s.dependency 'MMKVCore'
```

Comment:
dope - so if the example app adds a dependency to `MMKV` it can use that from ObjC just fine?